gooddata-sdk/gooddata_sdk/compute/service.py

```diff
@@ -91,21 +95,31 @@ def ai_chat(self, workspace_id: str, question: str) -> ChatResult:
         response = self._actions_api.ai_chat(workspace_id, chat_request, _check_return_type=False)
         return response
 
-    def ai_chat_history(self, workspace_id: str, chat_history_interaction_id: int = 0) -> ChatHistoryResult:
+    def get_ai_chat_history(
+        self, workspace_id: str, chat_history_interaction_id: str | None = None, thread_id_suffix: str | None = None
+    ) -> ChatHistoryResult:
         """
         Get chat history with AI in GoodData workspace.
 
         Args:
             workspace_id: workspace identifier
-            chat_history_interaction_id: collect history starting from this interaction id
+            chat_history_interaction_id: collect history starting from this interaction id. If None, complete chat history is returned.
+            thread_id_suffix: suffix to identify a specific chat thread. If provided, chat_history_interaction_id is ignored.
         Returns:
-            str: Chat history response
+            ChatHistoryResult: Chat history response containing interactions and other metadata
         """
-        chat_history_request = ChatHistoryRequest(chat_history_interaction_id=chat_history_interaction_id)
+
+        if chat_history_interaction_id is None:
+            chat_history_interaction_id = ""
+        if thread_id_suffix is None:
+            thread_id_suffix = ""
```

Comment:
Consider having an empty string default for `chat_history_interaction_id` and `thread_id_suffix` parameters.